infra/cifuzz/clusterfuzz_deployment.py

```diff
@@ -133,23 +132,23 @@ def download_corpus(self, target_name):
     return corpus_dir
 
   def _get_build_name(self):
-    return self.BASE_BUILD_NAME + self.config.sanitizer
+    return self.config.sanitizer + '-latest'
```

Comment:
I don't agree with this change. I don't think latest should be in the name because there can be multiple artifacts existing at once with the same name.

Reply:
>This assumption doesn't hold with the Git Filestore for example (or if we support buckets as storage).

I don't understand why not, could you provide an example please?

>This also leaves things open for us to store more builds in the future, if the need ever arises.

Why not change this assumption later if we need to, rather than make it now. Unless you think doing it now will save trouble migrating later.

Reply:
e.g. for Git (or a Bucket) store, an build "name" may be stored at the directory`builds/name`. This is global across all CI runs, unless we pipe through a unique CI run ID somehow (which I don't think we should do). 

Doing this small thing now (just a longer name) will save trouble migrating later. More importantly this is just setting a convention that artifacts have a name, and it's an implementation detail to either prepend a type prefix or a directory path when it's actually stored. 

i.e. GitHub artifacts -> upload_build("name") -> upload_directory("build-" + "name")
Git -> upload_build("name") -> upload_directory("build/" + "name")